### ISSUE TITLE
new version is buggy (in Chrome)

### ISSUE THREAD

**Issue by bugwelle** [ajoslin/angular-scrolly#12]
Hello,
I know that I'm reporting many issue right now, but this one is a bit annoying. :)

I've build the new version (see #11; I commented those lines out) and edited the demo, so it works with Angular 1.1.4 and the new angular-scrolly file (I only edited the script tags).

But now (with dev tools - touch enabled) scrolling is really buggy. When I try to scroll with my mouse (not the mouse wheel), it sometimes scrolls and sometimes it doesn't.

Maybe it's because of the latest commits. I haven't looked into them.
I hope someone could help.

Thanks,
Andre

**Comment by ajoslin** [comment]
Scrolling with the mouse on desktop is really unreliable - I wouldn't recommend it really.  Just use the mousewheel / arrows / etc.

**Comment by ajoslin** [comment]
Oh wait, do you mean dragging the scrollbar or dragging the content?

**Comment by bugwelle** [comment]
I mean dragging the content. I'm testing my mobile app in Chrome ;)
I don't mean scrolling with the mouse wheel. That works well. :)

**Comment by ajoslin** [comment]
OK.  Yeah, I dunno how to fix it lol, but I dont think most people would try it really.

**Comment by bugwelle** [comment]
I'll try it on my mobile device, but last version worked well in Chrome. 
If I find the reason for that buggy behaviour, I'll let you know ;)

**Comment by ajoslin** [comment]
OK. I removed the mouse drag events, that's probably why - dragging is touch only now.

**Comment by bugwelle** [comment]
removing this:

``` js
     //Hack taken from iscroll for mouse events
      elm.bind('mouseout', function mouseout(e) {
        e = e.originalEvent || e;
        var t = e.relatedTarget;
        if (!t) { 
          dragEnd(e);
        } else {
          while ( (t = t.parentNode) ) {
            if (t === elm) return;
          }
          dragEnd(e);
        }
      });
```

from `dragger.js` fixed this. :)

**Comment by ajoslin** [comment]
Nice :-)!

**Comment by ajoslin** [comment]
Fixed in b4daccd9913f2bfc40d37efc4b8b47fd5bbe0787
